cli/command_blob_shards_modify_test.go

```diff
@@ -20,38 +20,38 @@ func TestBlobShardsModify(t *testing.T) {
 
 	// verify default sharding is 3,3
 	require.FileExists(t, filepath.Join(env.RepoDir, someQBlob[0:3], someQBlob[3:6], someQBlob[6:]+sharded.CompleteBlobSuffix))
-	require.FileExists(t, filepath.Join(env.RepoDir, "kopia.repository.f"))
+	require.FileExists(t, filepath.Join(env.RepoDir, "kopia.repository"+sharded.CompleteBlobSuffix))
```

Comment:
i think it's ok to hardcode constants like that otherwise we're not ever sure what the filenames are asserted.

Reply:
ahh, that's very true and something i hadn't considered. alright, i'll keep this as my little branch to fiddle with and close this out. sorry for all the PRs lately 😅